### PR TITLE
solved problem 4.js/js0

### PR DESCRIPTION
**Pull request by gsstudent4**

### CODE REVIEW

File: curriculum/js0/4.js [hunk 1]
```diff
@@ -5,9 +5,14 @@
...
-  return 0;
-};
+    if (a>b) {
+  return a;
+    };
 
+if (b>a) {
+  return b;
+}
+};
```
Comment (gsstudent4) [inline curriculum/js0/4.js]:
```
if (a > b) return a;
else return b;
```

File: curriculum/js0/4.js [hunk 2]
```diff
@@ -5,9 +5,14 @@
  */
 
 const solution = (a, b) => {
-  return 0;
-};
+    if (a>b) {
+  return a;
```
Comment (roycep) [inline curriculum/js0/4.js]: Make sure that your indentation is neat and correct.

File: curriculum/js0/4.js (hunk 1 above)
Comment (songz) [reply curriculum/js0/4.js]: no need to use else.

Comment (seemcat) [reply curriculum/js0/4.js]: if (a > b) return a; 
return b;

**Review by seemcat — CHANGES_REQUESTED** [review]

**Review by roycep — APPROVED** [review]

**Pull request merged**